app/views/taxons/edit.html.erb

```diff
@@ -2,12 +2,20 @@
   breadcrumbs: [:taxons, link_to(taxon.internal_name, taxon_path(taxon.content_id)), "Edit"] %>
 
 <%= simple_form_for taxon, url: taxon_path(taxon.content_id), method: :patch do |f| %>
+  <% if flash[:error].present? %>
```

Comment:
I think if you use `flash[:danger]` a flash message will automatically appear:

https://github.com/alphagov/govuk_admin_template#flash-messages